install-acts-as-list

From: Bryan Larsen <bryan@larsen.st>

# Task re-ordering

We're now going to add the ability to re-order a story's tasks by
drag-and-drop. There's support for this built into Hobo, so there's
not much to do. First we need the `acts_as_list` plugin.  Add

    gem "acts_as_list"

to Gemfile and run

    $ bundle
---
 Gemfile      |    4 +++-
 Gemfile.lock |    2 ++
 2 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/Gemfile b/Gemfile
index ec53aa9..7f4301f 100644
--- a/Gemfile
+++ b/Gemfile
@@ -45,4 +45,6 @@ gem "will_paginate", :git => "git://github.com/Hobo/will_paginate.git"
 gem "hobo_bootstrap", "2.0.0.pre8"
 gem "hobo_jquery_ui", "2.0.0.pre8"
 gem "hobo_bootstrap_ui", "2.0.0.pre8"
-gem "jquery-ui-themes", "~> 0.0.4"
\ No newline at end of file
+gem "jquery-ui-themes", "~> 0.0.4"
+
+gem "acts_as_list"
diff --git a/Gemfile.lock b/Gemfile.lock
index 038b15b..db6dac0 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -34,6 +34,7 @@ GEM
     activesupport (3.2.11)
       i18n (~> 0.6)
       multi_json (~> 1.0)
+    acts_as_list (0.1.9)
     arel (3.0.2)
     bootstrap-datepicker-rails (0.6.35)
       railties (>= 3.0)
@@ -153,6 +154,7 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  acts_as_list
   coffee-rails (~> 3.2.1)
   hobo (= 2.0.0.pre8)
   hobo_bootstrap (= 2.0.0.pre8)
